INSTALL.md

```diff
@@ -21,6 +21,18 @@ If you have added new functions, you will need to add an explicit
 reference to them in
 [docs/study-def-variables.md](./docs/study-def-variables.md).
 
+
+### Updating Cohort Extractor v2 Definitions
+
+cohort-extractor-v2 is a documentation dependency which we can't install directly.
```

Comment:
suggestion: Maybe a one sentence explanation or link to relevant issue is useful here? I assume it's because of the legacy Python version on Cloudflare Pages.